<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoFaceCapture</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="autoCapture">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Auto Capture when face farmework still</string>
          </property>
          <property name="statusTip">
           <string>Auto Capture when face farmework still</string>
          </property>
          <property name="text">
           <string>&amp;Auto Capture</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/preference.png</normaloff>:/icons/preference.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+A</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>auto capture time</string>
          </property>
          <property name="statusTip">
           <string>auto capture time</string>
          </property>
          <property name="suffix">
           <string> sec</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="captureButton">
          <property name="toolTip">
           <string>Capture the image from camera</string>
          </property>
          <property name="statusTip">
           <string>Capture the image from camera</string>
          </property>
          <property name="text">
           <string>&amp;Capture</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/capture.png</normaloff>:/icons/capture.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+C</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="faceCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>If selected, the image captured will be cut.</string>
          </property>
          <property name="statusTip">
           <string>If selected, the image captured will be cut.</string>
          </property>
          <property name="text">
           <string>Use &amp;Face Framework</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/autoCapture.png</normaloff>:/icons/autoCapture.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="pathEditor">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>The floder of output file.</string>
          </property>
          <property name="statusTip">
           <string>The floder of output file.</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choseButton">
          <property name="toolTip">
           <string>Choose the oput floder.</string>
          </property>
          <property name="statusTip">
           <string>Choose the oput floder.</string>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/bower.png</normaloff>:/icons/bower.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="fileNameEdit">
          <property name="toolTip">
           <string>Name of the file to save with a counter(%1).</string>
          </property>
          <property name="statusTip">
           <string>Name of the file to save with a counter(%1).</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatBox">
          <property name="toolTip">
           <string>Format of the image being saved.</string>
          </property>
          <property name="statusTip">
           <string>Format of the image being saved.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="counter">
          <property name="toolTip">
           <string>A number counter</string>
          </property>
          <property name="statusTip">
           <string>A number counter</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="toolTip">
           <string>Reset the number counter</string>
          </property>
          <property name="statusTip">
           <string>Reset the number counter</string>
          </property>
          <property name="text">
           <string>&amp;Reset</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/reset.png</normaloff>:/icons/reset.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="statusTip">
     <string>File menu</string>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="statusTip">
     <string>Setting menu</string>
    </property>
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="statusTip">
     <string>About menu</string>
    </property>
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="mystatusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Set the margin and so on.</string>
   </property>
   <property name="statusTip">
    <string>Eixt this application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/setting.png</normaloff>:/icons/setting.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="toolTip">
    <string>Set the margin and so on.</string>
   </property>
   <property name="statusTip">
    <string>Set the margin and so on.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;bout</string>
   </property>
   <property name="toolTip">
    <string>About this application</string>
   </property>
   <property name="statusTip">
    <string>About this application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/aboutQt.png</normaloff>:/icons/aboutQt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
